.circleci/build_docs_mne.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Build near-minimal version of MNE-Python docs
+
+if [[ "$COLUMNS" == "" ]]; then COLUMNS=80; fi
+
+function __sep__ () {
+    echo
+    printf %"$COLUMNS"s | tr " " "-"
+    echo -e "\n"
```

Comment:
the function def is getting printed literally each time `__sep__` is called

https://app.circleci.com/pipelines/github/sphinx-gallery/sphinx-gallery/3082/workflows/568de77a-cd5f-440d-99e4-412eb1139104/jobs/7597?invite=true#step-104-38_94